### ISSUE TITLE
jekyll-relative-links doesn't update links in markdown

### ISSUE THREAD

**Issue by pro-src** [xtermjs/xtermjs.org#45]
The [official docs](https://help.github.com/articles/configuring-jekyll-plugins/#default-plugins) state that this [gem/plugin](https://github.com/benbalter/jekyll-relative-links) is a default plugin that cannot be disabled. However, it doesn't appear to work at all when deployed to github or locally.

From the _README.md_:

> ## What it does
> Let's say you have a link like this in a Markdown file: ``` [foo](bar.md) ```
> While that would render as a valid link on GitHub.com, it would not be a valid link on Pages.
> Instead, this plugin converts that link to: ``` [foo](bar.html) ```

It's supposed to make relative markdown links work without modification when deployed to gh-pages. Some of the things that could be handled by this plugin are currently being handled by the docs gulp task.

`foo.md` and the adjacent `bar.md` are converted to `foo/index.html` and `bar/index.html` respectively by the Jekyll build process.

A relative link in `foo.md` must refer to `bar/index.html` by `../../bar` thus the normal `../bar.md` and/or simply `bar.md` must be converted for links to work properly.

#### *TL;DR*
This link conversion process adds some (would be) unnecessary complexity to current build scripts. It would be better if we could get this plugin to work to simplify things.

Relevant Issues:
* https://github.com/github/pages-gem/issues/69
